src/components/MessageHistory/index.js

```diff
@@ -1,3 +1,6 @@
+/**
+ * Since this is a child of Client should it just move to be a child of the Client folder?
```

Comment:
With all of these, I did consider moving them to the client folder, but I hesitated because it meant adding another ../ to all the import paths. Also, this reminded me SO much of writing Perl modules because even though you're trying to modularize and componentize, you still end up with everything being inextricably coupled to everything else in the project because it has to know where those things it imports live. Ugh!